### PR TITLE
Update games.csv

### PR DESCRIPTION
**Pull request by xLeviNx**
Add Humanitz, Astro Colony, Icarus, Avorion to game list and query

### CODE REVIEW

**Review by xLeviNx — COMMENTED** [review]
@BattlefieldDuck - requesting a review

File: discordgsm/games.csv [hunk 1]
```diff
@@ -21,6 +21,8 @@ armareforger,Arma Reforger (2022),source,port=2001;port_query=17777
 asa,Ark: Survival Ascended (2023),asa,port=7777
 assettocorsa,Assetto Corsa (2014),assettocorsa,port=9610
 atlas,Atlas (2018),source,port=5761;port_query_offset=51800
+astrocolony,Astro Colony (2022),source,port_query=27015
+avorion,Avorion (2017),source,port=40017,port_query_offset=2
```
Comment (BattlefieldDuck) [inline discordgsm/games.csv:25]: should be `port=40017;port_query_offset=2`

Comment (xLeviNx) [reply discordgsm/games.csv]: ah good catch, fixed

**Review by xLeviNx — COMMENTED** [review]
fixed ;

**Pull request merged**